client/src/App.jsx

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Login from './components/Login';
 import './App.css';
+import SearchSidebar from './components/searchSidebar/sidebar';
```

Comment:
React component convention should follow the ```PascalCase```.
Also, the convention is to export the index.js/jsx.
```suggestion
import SearchSidebar from './components/SearchSidebar';
```